packages/ws-worker/package.json

```diff
@@ -13,6 +13,7 @@
     "start:watch": "nodemon -e ts,js --watch ../runtime-manager/dist --watch ./src --exec 'pnpm start'",
     "pack": "pnpm pack --pack-destination ../../dist"
   },
+  "bin": "NODE_OPTIONS=\"--experimental-vm-modules --no-warnings\" ts-node-esm --transpile-only src/start.ts",
```

Comment:
I didn't know you could inline a command like this in bin - pretty cool!

This may be a dumb question but why would we want `npx @openfn/ws-worker` rather than `npm start`?

I'd see npx as an end user convenience, making it super easy to download and run the worker. But I don't know who the user is who wants that?

Reply:
If it's for Lightning's convenience then I defer to @stuartc 

But if Lightning has a dev-time dependency on `ws-worker`, surely it's more transparent and honest to just declare it as a dev dependency?

The bin stub would still be useful though actually because Lightning will want to do `npm exec @openfn/ws--worker` or something to run it from the local repo